packages/libs/react-ui/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,36 @@
+import { ProductIcon } from '../Icons';
+
+import { containerClass } from './Breadcrumbs.css';
+import { BreadcrumbsItem, IBreadcrumbItemProps } from './BreadcrumbsItem';
+
+import React, { FC, FunctionComponentElement } from 'react';
+
+export interface IBreadcrumbsProps {
+  children?: FunctionComponentElement<IBreadcrumbItemProps>[];
+  icon?: typeof ProductIcon[keyof typeof ProductIcon];
+}
+
+export const BreadcrumbsContainer: FC<IBreadcrumbsProps> = ({
+  children,
+  icon,
+}) => {
+  return (
+    <div className={containerClass}>
```

Comment:
So breadcrumbs is used for navigation correct? Why aren't we using the nav item here?

Reply:
It looks like [the way to go](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)!

Instead of divs I've used the structure from the original breadcrumbs implementation (`<ul>`/`<li>`), and wrapped both in a `<nav>` tag

Reply:
Yes, and because we are using the Compound Components pattern it's ok to have a tight coupling of the `ul` and `li` structures here.